contracts/eosio.token/include/eosio.token/eosio.token.hpp

```diff
@@ -143,7 +143,7 @@ namespace eosio {
          static asset get_balance( const name& token_contract_account, const name& owner, const symbol_code& sym_code )
          {
             accounts accountstable( token_contract_account, owner.value );
-            const auto& ac = accountstable.get( sym_code.raw(), "no balance object found" );
+            const auto& ac = accountstable.get( sym_code.raw(), "Wrong symbol type" );
```

Comment:
This could be the "right" symbol type, except user has no balance

This would be a confusing error to receive

Reply:
"no balance with specified symbol" ?

Reply:
👍  that works